src/pages/administration/students/index.vue

```diff
@@ -30,31 +30,33 @@
 			@update:rows-per-page="onUpdateRowsPerPage"
 		>
 			<template v-slot:datacolumn-createdAt="{ data }">
-				{{ dayjs(data).format("DD.MM.YYYY") }}
+				<span class="text-content">{{ dayjs(data).format("DD.MM.YYYY") }}</span>
 			</template>
 			<template v-slot:datacolumn-consent="{ data }">
-				<span v-if="data && data.consentStatus === 'ok'">
-					<base-icon
-						source="custom"
-						icon="doublecheck"
-						color="var(--color-success)"
-					/>
+				<span class="text-content">
+					<template v-if="data && data.consentStatus === 'ok'">
+						<base-icon
+							source="custom"
+							icon="doublecheck"
+							color="var(--color-success)"
+						/>
+					</template>
+					<template v-else-if="data && data.consentStatus === 'parentsAgreed'">
+						<base-icon
+							source="material"
+							icon="check"
+							color="var(--color-warning)"
+						/>
+					</template>
+					<template v-else-if="data && data.consentStatus === 'missing'">
+						<base-icon
+							source="material"
+							icon="close"
+							color="var(--color-danger)"
+						/>
+					</template>
+					<template v-else />
```

Comment:
Now that I see this, it can be simplified even more.
1. the template tag can get removed and the condition can be directly at the according base-icon
2. the last v-else without content does nothing, it can be ommited.
```suggestion
				<span class="text-content">
					<base-icon
						v-if="data && data.consentStatus === 'ok'"
						source="custom"
						icon="doublecheck"
						color="var(--color-success)"
					/>
					<base-icon
						v-else-if="data && data.consentStatus === 'parentsAgreed'"
						source="material"
						icon="check"
						color="var(--color-warning)"
					/>
					<base-icon
						v-else-if="data && data.consentStatus === 'missing'"
						source="material"
						icon="close"
						color="var(--color-danger)"
					/>
```

Reply:
> Now that I see this, it can be simplified even more.
> 
> 1. the template tag can get removed and the condition can be directly at the according base-icon
> 2. the last v-else without content does nothing, it can be ommited.

ok, done :)